modules/ROOT/pages/comments-callback-mode.adoc

```diff
@@ -40,17 +41,14 @@ When using callback mode, the Comments plugin requires callback functions for th
 * xref:tinycomments_delete[`+tinycomments_delete+`]
 * xref:tinycomments_delete_all[`+tinycomments_delete_all+`]
 * xref:tinycomments_lookup[`+tinycomments_lookup+`]
-
-The xref:tinycomments_resolve[`+tinycomments_resolve+`] option is _optional_.
+* xref:tinycomments_fetch[`+tinycomments_fetch+`]
```

Comment:
This callback is optional as well. Current comments plugin users should not expect their implementation to break if they just upgrade.

CC: @Afraithe

Reply:
```suggestion
```